docs/tables/gitlab_my_event.md

```diff
@@ -0,0 +1,23 @@
+# Table: gitlab_my_event
+
+The `gitlab_my_event` table can be used to query information about your activity on a gitlab system.
+
+## Examples
+
+### Get all activity you're associated with
+```sql
+select
+  *

```

Comment:
Can we maybe list some key / prominent columns (doesn't need to be all), ideally we like to give user a sense of columns they can interface with that bring some immediate value as opposed to getting lost in the `*` - I am aware there are some other table docs which currently have `*` these are on a backlog to update.

Reply:
Maybe look at `/docs/tables/gitlab_group_project.md` or `/docs/tables/gitlab_project_access_request.d` for inspiration, I am aware other docs are in need of updating...

Reply:
Yep.  I'll do that now.  I had a third but it's slow w/o getIssuesByID. The activity api doesn't seem to set the `target_iid` for this - if it did we could use getIssue to do it. Really wish gitlab didn't have the id/iid thing going on.

Anyway, I'll come up with something.